generate-templates.yaml

```diff
@@ -35,6 +35,7 @@
       password: redhat
       oslabels: "{{ rhel8_labels }}"
       osinfoname: "{{ oslabels[0] }}"
+      version: 0.10.0
```

Comment:
Please do not do this here, it is not obvious when or how the version can be touched. I think it is much better to set the version in the template file itself. See my comment in the centos6 file.